privacyidea/api/validate.py

```diff
@@ -616,12 +617,22 @@ def poll_transaction(transaction_id=None):
                            if challenge.is_valid()]
     answered_challenges = extract_answered_challenges(matching_challenges)
 
+    decline_challenges = []
```

Comment:
Please name this "declined_challenges", since it contains the challenges, that have been declined.